examples/quick_start.ipynb

```diff
@@ -0,0 +1,260 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zDk-e7Pg_EdD"
+      },
+      "source": [
+        "# Quick start\n",
+        "\n",
+        "`optax` is a simple optimization library for Jax. The main object is the `GradientTransformation` which can be chained\n",
+        "with other transformations to obtain the final update operation and optimizer state.\n",
+        "`optax` also contains some simple loss functions and utilities to help you write the full optimization steps. This colab walks through a few examples on how to use optax to train a simple linear model:\n",
```

Comment:
Suggestion: I think it's Optax (name) vs package name optax (https://github.com/deepmind/optax/blob/master/README.md)

```suggestion
        "# Optax quickstart\n",
        "\n",
        "Optax is a simple optimization library for [JAX](https://jax.readthedocs.io/). The main object is the `GradientTransformation`, which can be chained\n",
        "with other transformations to obtain the final update operation and the optimizer state.\n",
        "Optax also contains some simple loss functions and utilities to help you write the full optimization steps. This notebook/Colab walks you through a few examples on how to use Optax to train a simple linear model. Begin with importing the necessary packages:\n",
```